src/components/post/PriceOfferCard/index.tsx

```diff
@@ -1,102 +1,188 @@
-import { Divider, ToggleButton, SelectBox, Text } from '@offer-ui/react'
+import { Divider, SelectBox, Text, Icon } from '@offer-ui/react'
 import type { ReactElement } from 'react'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import { Styled } from './styled'
 import type { PriceOfferCardProps } from './types'
+import { PriceOfferModal } from '../PriceOfferModal'
+import type { OfferForm } from '../PriceOfferModal/types'
 import { UserProfile } from '../UserProfile'
+import { useUpdateLikeStatusMutation } from '@apis/like'
+import { useGetPostOffersQuery, useCreateOfferMutation } from '@apis/offer'
+import { useGetPostQuery } from '@apis/post'
+import { SORT_OPTIONS } from '@constants/app'
+import { getTimeDiffText, toLocaleCurrency } from '@utils/format'
+import { useModal } from '@hooks'
+import type { SortOption, SortOptionCodes } from '@types'
 
 const PriceOfferCard = ({
-  offerList,
-  likeCount,
-  isLike
+  postId,
+  isSeller
 }: PriceOfferCardProps): ReactElement => {
-  const hasOffer = offerList.length > 0
-  const [isLikeToggle, setIsLikeToggle] = useState<boolean>(isLike)
+  const [sortOption, setSortOption] = useState<SortOptionCodes>(
+    SORT_OPTIONS[0].code
+  )
+  const {
+    isOpen: isOfferModalOpen,
+    openModal: openOfferModal,
+    closeModal: closeOfferModal
+  } = useModal()
+  const [likePost, setLikePost] = useState({
+    status: false,
+    count: 0
+  })
+
+  const postOffersQuery = useGetPostOffersQuery({ postId, sort: sortOption })
+  const postQuery = useGetPostQuery(postId)
+  const likeStatusMutation = useUpdateLikeStatusMutation()
+  const offerMutation = useCreateOfferMutation()
+
+  useEffect(() => {
+    setLikePost({
+      status: Boolean(postQuery.data?.liked),
+      count: postQuery.data?.totalLikeCount || 0
+    })
+  }, [postQuery.isSuccess])
+
+  const offers =
+    postOffersQuery.data?.offers.map(({ id, offerer, createdAt, price }) => ({
+      ...offerer,
+      level: Number(offerer.level),
+      id,
+      date: createdAt,
+      price
+    })) || []
```

Comment:
```suggestion
    postOffersQuery.data?.offers.map(({offerer, createdAt }) => ({
      ...offerer,
      level: Number(offerer.level),
      date: createdAt,
    })) || []
```
전개 연산자만으로도 충분할 것 같습니다!

Reply:
[완료](https://github.com/price-offer/offer-fe/pull/203/commits/f4418950e2c876977d9d0097c520cd0746d2f0f0#diff-99868c718a54a36948d73a8b379e24547ec16c2d4007dd4b2675768b9b540c16R49-R53)